### PR TITLE
chore: Hide tabs on send when reading QR code

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-516](https://emurgo.atlassian.net/browse/YOMO-516)

[YOMO-516]: https://emurgo.atlassian.net/browse/YOMO-516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/i18n/locales/en-US.json [hunk 1]
```diff
@@ -131,7 +131,7 @@
   "components.send.assetselectorscreen.youHave": "You have",
   "components.send.assetselectorscreen.noAssetsAddedYet": "No {fungible} added yet",
   "components.send.addressreaderqr.title": "Scan QR code address",
-  "components.send.addressreaderqr.text": "Scan recipients QR code to add a wallet address",
+  "components.send.addressreaderqr.text": "Scan recipients QR code to add a crypto address",
```
Comment (banklesss) [inline apps/wallet-mobile/src/i18n/locales/en-US.json]: restored until Product team confirmation

File: apps/wallet-mobile/src/components/QRCodeScanner/QRCodeScanner.tsx [hunk 2]
```diff
@@ -182,7 +182,7 @@ export const getScaledQrBounds = ({
   deviceHeight: number
   deviceWidth: number
 }) => {
-  // qr bounds values are inversely proportioned + issues in some android devices https://github.com/expo/expo/issues/17795
```
Comment (banklesss) [inline apps/wallet-mobile/src/components/QRCodeScanner/QRCodeScanner.tsx:185]: that bug its fixed in latest version. Also, the qr position data is not optional anymore, as the comments in that issue suggest

File: apps/wallet-mobile/src/i18n/locales/en-US.json (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/i18n/locales/en-US.json]: confirmed

File: apps/wallet-mobile/src/features/Send/common/strings.ts [hunk 3]
```diff
@@ -253,6 +253,6 @@ export const messages = defineMessages({
   },
   addressReaderQrText: {
     id: 'components.send.addressreaderqr.text',
-    defaultMessage: '!!!Scan recipients QR code to add a crypto address',
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Send/common/strings.ts:256]: missing change

File: apps/wallet-mobile/src/WalletNavigator.tsx [hunk 4]
```diff
@@ -50,7 +51,14 @@ const WalletTabNavigator = () => {
...
             tabBarLabel: strings.walletTabBarLabel,
             tabBarTestID: 'walletTabBarButton',
-          }}
+            tabBarStyle: (({route}) => {
+              const routeName = getFocusedRouteNameFromRoute(route) ?? ''
+              if (routeName === 'send-read-qr-code') {
+                return {display: 'none'}
+              }
+              return
+            })(route),
```
Comment (michalszorad) [inline apps/wallet-mobile/src/WalletNavigator.tsx]: Is it needed to call this function here?

Comment (banklesss) [reply apps/wallet-mobile/src/WalletNavigator.tsx]: `tabBarStyle` type is `undefined` or an `object` and i need the `route` from `options` callback and the `routeName` from `getFocusedRouteNameFromRoute`

Let me know if you have any suggestion!

Comment (michalszorad) [reply apps/wallet-mobile/src/WalletNavigator.tsx]: @jemubm do you think `tabBarStyle: getFocusedRouteNameFromRoute(route.route) === 'send-read-qr-code' ? {display: 'none'} : {},` could work?

Comment (banklesss) [reply apps/wallet-mobile/src/WalletNavigator.tsx]: If more routes are required in which to hide the tabs, a slight change will have to be made but it works and I have applied it

thank you!

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx [hunk 5]
```diff
@@ -142,6 +142,7 @@ export const TxHistoryNavigator = () => {
           name="send-read-qr-code"
           component={ReadQRCodeScreen}
           options={{
+            // tabs removed from this screen route in wallet tab navigator
```
Comment (stackchain) [inline apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx]:
```suggestion
```

**Pull request merged**